src/api/restaurants/restaurants.services.ts

```diff
@@ -165,3 +165,16 @@ export const deleteRestaurant = (id: string) => {
     },
   });
 };
+
+//update restaurant rating
+
+export const updateRestaurantRating = (id_restaurant: any, rating: any) => {
```

Comment:
why not datatype for params?

Reply:
I am still not very clear about the typing, I would like us to meet one day to understand this issue and clean up the code